layouts/page/cudf-polars.html

```diff
@@ -0,0 +1,441 @@
+{{- define "main" -}}
+
+<!-- Hero Partial -->
+{{ partial "hero.html" (dict
+"text" "Polars GPU engine powered by cuDF"
+"btn_text" "Get Started"
+"btn_link" "#get-started"
+) }}
+
+<!-- Intro -->
+<section class="row section mb-6">
+  <div class="col">
+    <div class="container">
+
+      <h1 class="text-center mb-10"> BRINGING ACCELERATED COMPUTING TO POLARS <br> [Open Beta] </h1>
+      <div class="row">
+        <div class="col-md-4 py-3">
+          <h2><i class="fa-light fa-file-binary"></i> Zero Code Change</h2>
+          <p>Pass <code>engine=”gpu”</code> to the Polars' collect API to add GPU acceleration to your data processing tasks like data aggregation, merge, window functions, and much more.</p>
+
+          <br>
+
+          <h2 class="mt-4"><i class="fa-sharp fa-light fa-solar-system"></i> Ecosystem Compatible</h2>
+          <p>Polars' growing ecosystem of data visualization, IO, & ML libraries can be accelerated with the GPU engine.</p>
+        </div>
+        <div class="col-md-4 py-3">
+          <h2><i class="fa-solid fa-light fa-microchip"></i> Seamless CPU Fallback</h2>
+          <p>For queries where one or more operations are not supported in cuDF, the entire query execution will gracefully fallback to the default CPU engine.</p>
+
+          <br>
+
+          <h2 class="mt-4"><i class="fa-light fa-bolt-lightning"></i> Query Optimizer</h2>
+          <p>The GPU engine utilizes Polars' query optimizer to ensure efficient execution and minimal memory usage.</p>
+        </div>
+        <div class="col-md-4 py-3">
+          <iframe width="550" height="300" src="https://www.youtube.com/embed/AoKeit2Fbmw?si=Gin-Bwr5ndvaMABI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+        </div>
+      </div>
+
+    </div>
+  </div>
+</section>
+
+<div class="bottom-slant-container">
+  <div class="bottom-slant-down bg-white"></div>
+</div>
+
+<!-- Charts -->
+<section class="row section padding-top-slant">
+  <div class="col">
+    <div class="container">
+
+      <h1 class="text-center mb-4"> Process 100s of Millions of Rows in Seconds </h1>
```

Comment:
100s is appearing as 100S. If it defaults to all caps, can we use 'HUNDREDS'

PROCESS HUNDREDS OF MILLIONS OF ROWS IN SECONDS